var/spack/repos/builtin/packages/jsonnet/package.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class Jsonnet(MakefilePackage):
+    """A data templating language for app and tool developers based on JSON"""
+
+    homepage = "https://jsonnet.org/"
+    git = "https://github.com/google/jsonnet.git"
+
+    maintainers = ["jcpunk"]
+
+    version("develop", branch="master")
+    version("0.18.0", branch="v0.18.0")
+    version("0.17.0", branch="v0.17.0")
```

Comment:
I don't see a v0.19.0 branch, just a tag. Also, Can we pin the commit she instead?